docs/development/workflow/virtual_pythons.rst

```diff
@@ -133,7 +133,7 @@ environment.
       in order for the development version to install properly. You can do this
       with the following command::
 
-        conda uninstall astropy
+        conda uninstall astropy -n ENV
```

Comment:
Huh, is this really necessary? Do people often uninstall stuff from a different env?

Reply:
I was following the order of the documentation.  The activation of the environment does not happen until later.  Thus, after creating the environment, I was still in `base`.  This is intended to prevent accidentally uninstalling a stable astropy installation.

Reply:
Interesting... I *think* the assumption is that by the time you get to the point you need to uninstall, you would already be in that environment because the second bullet points to other links. I agree that it is not an ideal assumption but I would rather we add a `conda active ENV` after the creation command, than to add `-n ENV` here.

Reply:
I agreed.  I'll implement a fix to ensure the activation.